packages/noya-renderer/src/components/SketchFileRenderer.tsx

```diff
@@ -451,6 +454,26 @@ export default memo(function SketchFileRenderer() {
     );
   }, [isEditingPath, page, state]);
 
+  const symbol = useMemo(() => {
+    if (interactionState.type !== 'insertingSymbol') return;
+    // TODO: Modifying only the instance of the symbol master, not the master itself. Improve mouse pointer position.
+    const symbol = {
+      ...getSymbols(state).find(
+        ({ do_objectID }) => do_objectID === interactionState.symbolID,
+      ),
+    } as Sketch.SymbolMaster;
+
+    if (!symbol || !symbol.style) return;
+    symbol.style.contextSettings = {
```

Comment:
We can avoid modifying the original symbol master via `produce`:

```ts
produce(symbol, (draft) => {
  symbol.style.contextSettings = ...
})
```

Reply:
ohh! I'll do that